proxy/http/HttpSM.cc

```diff
@@ -3909,29 +3904,6 @@ HttpSM::tunnel_handler_plugin_agent(int event, HttpTunnelConsumer *c)
   return 0;
 }
 
-int
-HttpSM::state_srv_lookup(int event, void *data)
-{
-  STATE_ENTER(&HttpSM::state_srv_lookup, event);
-
-  ink_assert(t_state.req_flavor == HttpTransact::REQ_FLAVOR_SCHEDULED_UPDATE ||
-             t_state.req_flavor == HttpTransact::REQ_FLAVOR_REVPROXY || ua_entry->vc != nullptr);
-
-  switch (event) {
-  case EVENT_SRV_LOOKUP:
-    pending_action = nullptr;
```

Comment:
Did we remove some other code that used this? Or is there new code dealing with these events?

Reply:
My guess is that logic moved into HttpSM::state_hostdb_lookup.  The HostDB will be calling back to the HttpSM contination with the SRV_LOOKUP event and state_hostdb_lookup is still being set as a handler.